src/SimplCommerce.WebHost/gulpfile.js

```diff
@@ -83,9 +83,9 @@ gulp.task('clean-module', function () {
 gulp.task('copy-static', function () {
     modules.forEach(function (module) {
         console.log('copying static contents ' + paths.dev.modules + module.fullName);
-        gulp.src([paths.dev.modules + module.fullName + '/Views/**/*.*',
-        paths.dev.modules + module.fullName + '/module.json'], { base: module.fullName })
-            .pipe(gulp.dest(paths.host.modules + module.fullName));
+        //gulp.src([paths.dev.modules + module.fullName + '/Views/**/*.*',
```

Comment:
please tell my why you have to comment this

Reply:
In the new version of ASP.NET Core, when we compile a project it renders two files including **{ProjectName}.Views.dll ** where the Razor views reside and by referencing that project in another project, we can use the views as well like we use other C# classes. IMO, that's why it's not needed to copy the views using _gulp_. It also increases build performance and make the deployment more light-weight

Reply:
Have you tested it?

Reply:
Yes I did. It uses Razor Class Library (RCL).
FYI [https://docs.microsoft.com/en-us/aspnet/core/razor-pages/sdk?view=aspnetcore-2.1](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/sdk?view=aspnetcore-2.1)

Reply:
Just checkout the pull request. It's not work on my laptop. The error is that it cannot find the views. The same as my try in some months ago. Because the modular architecture and the the customization of ViewLocationExpander